pages/_document.js

```diff
@@ -0,0 +1,24 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+// Used to augment application's html and body tags
+// Read more here: https://nextjs.org/docs/advanced-features/custom-document
+class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  render() {
+    return (
+      <Html lang="en">
```

Comment:
This is so that we can add the `lang="en"`, right?

Reply:
Yes to modify those attributes you have to create a custom Document "component" to override the Next default.

https://nextjs.org/docs/advanced-features/custom-document